core/src/main/java/org/apache/iceberg/CachingCatalog.java

```diff
@@ -78,7 +83,9 @@ public Table loadTable(TableIdentifier ident) {
         TableOperations ops = ((HasTableOperations) originTable).operations();
         MetadataTableType type = MetadataTableType.from(canonicalized.name());
 
-        Table metadataTable = MetadataTableUtils.createMetadataTableInstance(ops, originTableIdentifier, type);
+        Table metadataTable = MetadataTableUtils.createMetadataTableInstance(
+            ops, name(), originTableIdentifier,
```

Comment:
Minor: Although it is the same thing, I think it would make this more readable to use `catalog.name()` directly. That way it is clear that the name isn't coming from the caching catalog.

Reply:
Done.